src/pages/public/about-us-page/intro-section/IntroSection.tsx

```diff
@@ -7,17 +7,15 @@ export const AboutUsIntro = () => {
         <div className="about-us-block">
             <img src={background} className="background-img" alt="Men and Horse" />
             <img src={background} className="color-overlay" alt="Men and Horse" />
```

Comment:
I understand that's not your changes, but do me a favor, please :)
A screen reader will read the alt text "Men and Horse" twice, creating a confusing and redundant experience. This adds an unnecessary `img` element to the DOM. A color overlay should never be an image - it's a styling concern.

> [!IMPORTANT] 
> The color overlay must be implemented in CSS using a pseudo-element on the parent container

The alt text "Men and Horse" describes the image, but this image serves a purely decorative/background purpose. Its content is not relevant to understanding the page. Screen readers should skip it and read the `h1` title first.

BTW, is it possible to completely remove `img` tag and use only styles? `background-image`, `background-position`, `background-size`, etc...
```suggestion
        <section className="about-us-block">
            <img src={background} className="background-img" alt="" />
```